lib/tasks/whitehall_importer.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module Tasks
+  class WhitehallImporter
+    SUPPORTED_WHITEHALL_STATES = %w(draft published rejected submitted superseded).freeze
+    SUPPORTED_LOCALES = %w(en).freeze
+
+    def import(whitehall_document)
+      ActiveRecord::Base.transaction do
+        @most_recent_edition = most_recent_edition(whitehall_document)
+        @document = create_or_update_document(whitehall_document)
```

Comment:
It's rather unusual for an method to be setting instance variables that apply just for the duration of that methods running, I'd suggest this indicates that these are items that the class should be initialized with and then set to `attr_reader`

Reply:
Refactored this class to move the instance variables into an initialize method